src/vs/workbench/browser/parts/panel/panelPart.ts

```diff
@@ -94,12 +94,13 @@ export class PanelPart extends CompositePart<Panel> implements IPanelService {
 			hidePart: () => this.partService.setPanelHidden(true),
 			compositeSize: 0,
 			overflowActionSize: 44,
-			colors: {
-				backgroundColor: PANEL_BACKGROUND,
-				badgeBackground,
-				badgeForeground,
-				dragAndDropBackground: PANEL_DRAG_AND_DROP_BACKGROUND
-			}
+			colors: theme => ({
+				backgroundColor: theme.getColor(PANEL_BACKGROUND),
+				activeBackgroundColor: theme.getColor(PANEL_BACKGROUND),
```

Comment:
@sandy081 I know this might have been like that before but I find the term "backgroundColor" and "activeBackgroundColor" misleading because we are actually setting the foreground color, are we? Maybe I started to call it background color because technically it will end up to be the background-fill color of the activity icon, but I think it should rather be changed to "foreground" to make it easier to understand.

Reply:
Composite bar colours are used by both Activity bar and Panel. `BackgroundColor` is foreground colour for activity bar but background colour for panel. So changing it to foreground colour will break panels code. So I will leave it to be background colour atleast it matches to what we set at the end.

Reply:
@sandy081 are you sure? I see a color `panelTitle.activeForeground` which seems to be doing the same thing:

![image](https://user-images.githubusercontent.com/900690/45479126-ece72d80-b745-11e8-8cdf-cac7fe2154cf.png)

It is a bit unfortunate that we introduced inactive & active color for the panel.

Reply:
Foreground colour for the panel is not getting applied in the action instead it is applied outside in css (see below comment). I moved this to composite action. 

Also previously panel is sending background colour to the composite action which is not getting applied at all. This caused the confusion. Panel items and Activity bar items have only foreground colours.

I cleaned up the code and let composite bar action takes only foreground colours. Composite action applies foreground colours correctly for icons and labels.